enterprise/frontend/src/metabase-enterprise/whitelabel/components/MetabaseLinksToggleWidget/MetabaseLinksToggleDescription.tsx

```diff
@@ -0,0 +1,30 @@
+import { jt } from "ttag";
+import { Anchor, Popover, Stack, Text } from "metabase/ui";
+
+export function MetabaseLinksToggleDescription() {
+  return jt`Control the visibility of links to Metabase documentation and Metabase references in your instance. ${(
+    <Popover key="popover" position="top-start">
+      <Popover.Target>
+        <Anchor style={{ userSelect: "none", cursor: "pointer" }}>
+          Learn more
```

Comment:
Should these be wrapped in t?

Reply:
Yeah, I missed wrapping almost every copy in this file a t tag. Good catch!